packages/multichain/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0]
+
+### Changed
+
+- Bump @metamask/utils from ^11.1.0 to ^11.2.0 ([#5301](https://github.com/MetaMask/core/pull/5301))
```

Comment:
Nit: can you wrap package name and versions with backtick like line 25 in this file